resotolib/resotolib/baseresources.py

```diff
@@ -292,6 +290,10 @@ def clean(self, value: bool) -> None:
     def cleaned(self) -> bool:
         return self._cleaned
 
+    @property
+    def collected(self) -> bool:
```

Comment:
Impact: every collect will mark every resource as changed. Wouldn't it be sufficient to have this date on account level?